services/src/main/java/org/keycloak/services/clientpolicy/executor/ConfidentialClientAcceptExecutor.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.services.clientpolicy.executor;
+
+import org.keycloak.OAuthErrorException;
+import org.keycloak.models.ClientModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.services.clientpolicy.ClientPolicyContext;
+import org.keycloak.services.clientpolicy.ClientPolicyException;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * @author <a href="mailto:takashi.norimatsu.ws@hitachi.com">Takashi Norimatsu</a>
+ */
+public class ConfidentialClientAcceptExecutor implements ClientPolicyExecutorProvider<ClientPolicyExecutorConfiguration> {
+
+    protected final KeycloakSession session;
+
+    public ConfidentialClientAcceptExecutor(KeycloakSession session) {
+        this.session = session;
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class Configuration {
```

Comment:
It seems that this class is not needed at all - we can probably stick with the ClientPolicyExecutorConfiguration?

Just noticed that the same approach can be done on few other executor implementations (like for example SecureSessionEnforceExecutor) where the inner "Configuration" class can also be removed. It should also help to simplify some stuff in AbstractClientPoliciesTest, which can just use the generic ClientPolicyExecutorConfiguration for executors, which don't have any configuration options. WDYT?

Reply:
@mposolda Yes, we need not prepare `Configuration` for the executor which do not have any configuration.
In `AbstractClientPoliciesTest`, I would like to remove the configuration methods for such executors.